### PR TITLE
Add Pig mapper with unit and integration tests

### PR DESCRIPTION
**Pull request by sprzedwojski**

### CODE REVIEW

File: oozie-to-airflow/tests/mappers/test_pig_mapper.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+        name1.text = 'mapred.job.queue.name'
+        value1.text = '${queueName}'
+        name2.text = 'mapred.map.output.compress'
+        value2.text = 'false'
+
+        param1.text = 'INPUT=/user/${wf:user()}/${examplesRoot}/input-data/text'
+        param2.text = 'OUTPUT=/user/${wf:user()}/${examplesRoot}/output-data/demo/pig-node'
+
+        # ssh = ET.Element('ssh')
```
Comment (potiuk) [inline oozie-to-airflow/tests/mappers/test_pig_mapper.py]: Should we remove it ?

File: oozie-to-airflow/tests/mappers/test_pig_mapper.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        #                               params=params)
+        # # make sure everything is getting initialized correctly
+        # self.assertEqual('test_id', mapper.task_id)
+        # self.assertEqual(TriggerRule.DUMMY, mapper.trigger_rule)
+        # self.assertEqual(self.et.getroot(), mapper.oozie_node)
+        # self.assertEqual('user', mapper.user)
+        # self.assertEqual('apache.org', mapper.host)
+        # self.assertEqual('\'ls -l -a\'', mapper.command)
+
+    def test_convert_to_text(self):
```
Comment (potiuk) [inline oozie-to-airflow/tests/mappers/test_pig_mapper.py]: Same here

File: oozie-to-airflow/tests/mappers/test_pig_mapper.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        pass
+        # mapper = ssh_mapper.SSHMapper(oozie_node=self.et.getroot(),
+        #                               task_id='test_id',
+        #                               trigger_rule=TriggerRule.DUMMY)
+        # # Throws a syntax error if doesn't parse correctly
+        # ast.parse(mapper.convert_to_text())
+
+    def test_required_imports(self):
```
Comment (potiuk) [inline oozie-to-airflow/tests/mappers/test_pig_mapper.py]: And here

**Comment by sprzedwojski** [comment]
These commits became included in https://github.com/GoogleCloudPlatform/cloud-composer/pull/21. Closing.

**Pull request closed without merging**